src/libcore/fmt/mod.rs

```diff
@@ -180,8 +180,9 @@ enum Void {}
            issue = "0")]
 #[doc(hidden)]
 pub struct ArgumentV1<'a> {
-    value: &'a Void,
-    formatter: fn(&Void, &mut Formatter) -> Result,
+    _ph: PhantomData<&'a ()>,
```

Comment:
I don't think there was a match anywhere. Just something more to do with having an uninhabited type in live code causing segfaults or something.